wav_in

```diff
@@ -5,3 +5,32 @@
   dtmin = 50.
   initfile = "./input/restart.ww3"
 /
+&input_nml
+  input%forcing%water_levels = 'T'
+  input%forcing%currents     = 'C'
+  input%forcing%winds        = 'C'
+  input%forcing%ice_conc     = 'C'
+  input%forcing%ice_param1   = 'C'
+  input%forcing%ice_param5   = 'C'
+/
+&output_type_nml
+  type%field%list = 'WND ICE HS T02 T0M1 T01 FP DIR EF USS TOC IC1 IC5'
+  type%point%file = 'ww3_points.list'
+/
+&output_date_nml
+  date%field%outffile  = '1'
+  date%field%start     = '19000101 0'
+  date%field%stride    = '86400'
+  date%field%stop      = '99990101 0'
+  date%point%outffile  = '0'
+  date%point%start     = '19000101 0'
+  date%point%stride    = '0'
```

Comment:
Sorry, noticed one last thing. In IAF, this is `86400`. I suspect this doesn't matter since this is for point output, but maybe worth being consistent anyway?

Reply:
Thanks @dougiesquire for pointing this out, it should be zero so that the binary point output files will not be generated. I will open a separate PR for IAF config to change this to zero.